upi/azure/05_masters.json

```diff
@@ -70,7 +70,8 @@
     "internalLoadBalancerName" : "[concat(parameters('baseName'), '-internal-lb')]",
     "sshKeyPath" : "/home/core/.ssh/authorized_keys",
     "identityName" : "[concat(parameters('baseName'), '-identity')]",
-    "imageName" : "[concat(parameters('baseName'), '-image')]",
+    "galleryName": "[concat('gallery_', replace(parameters('baseName'), '-', '_'))]",
+    "imageName" : "[parameters('baseName')]",
```

Comment:
how about use the previous image name "[concat(parameters('baseName'), '-image')]" ?

Reply:
We can but then we also need to update https://github.com/openshift/installer/blob/master/pkg/asset/machines/azure/machines.go#L120. I made it consistent with the terraform changes https://github.com/openshift/installer/blob/master/data/data/azure/vnet/main.tf#L101-L130 but have no strong feelings about it.